socket.go

```diff
@@ -257,6 +257,15 @@ func (sck *socket) Type() SocketType {
 	return sck.typ
 }
 
+// Addr returns the listen address
```

Comment:
please adapt with whatever updated `zmq4.Socket.Addr` documentation you came up with.